src/main/java/graphql/schema/AbstractGraphQLType.java

```diff
@@ -0,0 +1,32 @@
+package graphql.schema;
+
+import static graphql.Assert.assertNotNull;
+
+public class AbstractGraphQLType implements GraphQLType {
+    private final String name;
+
+    public AbstractGraphQLType(String name) {
+        assertNotNull(name, "name can't null");
+        this.name = name;
+    }
+
+    public String getName() {
+        return name.equals("") ? null : name;
+    }
```

Comment:
We talked about this, the real question is here, if we have to return null values.